src/main/java/edu/wpi/grip/ui/ExceptionView.java

```diff
@@ -0,0 +1,191 @@
+    /**
+     * @see <a href="http://code.makery.ch/blog/javafx-dialogs-official/">Inspiration</a>
+     * @param throwable The throwable exception to display this alert for.
+     * @param services The host services, allows access to {@link HostServices#showDocument(String)} in order to display
+     *                 the issue website.
+     */
+    public ExceptionView(final Throwable throwable, final HostServices services) {
+        super(AlertType.ERROR);
+        checkNotNull(throwable, "The Throwable can not be null");
+        checkNotNull(services, "HostServices can not be null");
+
+        this.exceptionMessage = generateExceptionMessage(throwable);
+        this.systemInfoMessage = generateSystemInfoMessage();
+        this.initialCause = generateInitialCause(throwable);
+
+        this.setTitle(initialCause.getClass().getSimpleName());
+        this.setHeaderText(initialCause.getMessage());
+
+        this.getButtonTypes().addAll(copyToClipboardBtnType, openGitHubIssuesBtnType);
+
+        final Label whatHappenedLabel = new Label(ISSUE_PROMPT_QUESTION);
+
+        final TextArea issueText = new TextArea(issueText(""));
+
+        final TextArea inputBox = new TextArea();
+        inputBox.textProperty().addListener((observable, oldValue, newValue) -> issueText.setText(issueText(newValue)));
+        inputBox.setPromptText("Short description of what you were doing when this dialog appeared.");
+        inputBox.setEditable(true);
+        inputBox.setWrapText(true);
+        inputBox.setMaxWidth(Double.MAX_VALUE);
+        inputBox.setMaxHeight(Double.MAX_VALUE);
+        GridPane.setVgrow(inputBox, Priority.ALWAYS);
+        GridPane.setHgrow(inputBox, Priority.ALWAYS);
+
+        final GridPane messageContent = new GridPane();
+        messageContent.setMaxWidth(Double.MAX_VALUE);
+        messageContent.setMaxHeight(Double.MAX_VALUE);
+        messageContent.add(whatHappenedLabel, 0, 0);
+        messageContent.add(inputBox, 0, 1);
+        this.getDialogPane().setContent(messageContent);
+
+        final Label issuePasteLabel = new Label("Please paste this into an issue on the project GitHub:");
+
+        issueText.setEditable(false);
+        issueText.setWrapText(true);
+
+        issueText.setMaxWidth(Double.MAX_VALUE);
+        issueText.setMaxHeight(Double.MAX_VALUE);
+        GridPane.setVgrow(issueText, Priority.ALWAYS);
+        GridPane.setHgrow(issueText, Priority.ALWAYS);
+
+        final GridPane expContent = new GridPane();
+        expContent.setMaxWidth(Double.MAX_VALUE);
+        expContent.add(issuePasteLabel, 0, 0);
+        expContent.add(issueText, 0, 1);
+
+        // Set expandable Exception into the dialog pane.
+        this.getDialogPane().setExpandableContent(expContent);
+        this.getDialogPane().setExpanded(true);
+
+
+        // Prevent these two buttons from causing the alert to close
+        final Button copyToClipboardBtn = (Button) this.getDialogPane().lookupButton(copyToClipboardBtnType);
```

Comment:
TextArea lets you right click and copy, and our users will certainly be computer literate enough to know that